gensim/models/doc2vec.py

```diff
@@ -917,27 +917,24 @@ def infer_vector(self, doc_words, alpha=None, min_alpha=None, epochs=None, steps
         doctag_vectors, doctag_locks = self.trainables.get_doctag_trainables(doc_words, self.docvecs.vector_size)
         doctag_indexes = [0]
         work = zeros(self.trainables.layer1_size, dtype=REAL)
+        neu1 = None
```

Comment:
Why did you add `neu1` keyword parameter? Why are you ignoring its specified value and setting it to None?